pages/in/civic/index.vue

```diff
@@ -1,10 +1,19 @@
 <template>
-  <div>{{$t('General.loading')}}</div>
+  <div class="lc-container-0">
+    <div class="lc-container-1">
+      <div class="lc-container-2">
+        <div class="lc-container-3">
+          <lc-loading-indicator />
+        </div>
+      </div>
+    </div>
+  </div>
```

Comment:
@nwingt please comment this, I don't know why the containers are necessary

Reply:
Updated